frame/examples/split/src/mock.rs

```diff
@@ -0,0 +1,59 @@
+use crate as pallet_template;
+use frame_support::traits::{ConstU16, ConstU64};
+use sp_core::H256;
+use sp_runtime::{
+	testing::Header,
+	traits::{BlakeTwo256, IdentityLookup},
+};
+
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+// Configure a mock runtime to test the pallet.
+frame_support::construct_runtime!(
+	pub enum Test where
+		Block = Block,
+		NodeBlock = Block,
+		UncheckedExtrinsic = UncheckedExtrinsic,
+	{
+		System: frame_system,
+		TemplateModule: pallet_template,
```

Comment:
The keyword module is wrong, it should be called `Pallet`. 

This is exactly why I am saying that template pallets should be up to date, and utmost clean. Even internally we copy paste them around and if there is something wrong in one, it spreads like a virus :D